collector/pg_stat_statements.go

```diff
@@ -71,10 +83,20 @@ var (
 		prometheus.Labels{},
 	)
 
+	statStatementsQuery = prometheus.NewDesc(
+		prometheus.BuildFQName(namespace, statStatementsSubsystem, "query_id"),
+		"SQL Query to queryid mapping",
+		[]string{"queryid", "query"},
+		prometheus.Labels{},
+	)
+
+	pgStatStatementQuerySelect = "LEFT(pg_stat_statements.query, 120) as query,"
```

Comment:
I think it would be worth adding a flag for the length.

Also, this should be a `const`, not a `var` if it's a static string template.

Reply:
Done